shared/AppInsightsCommon/src/Interfaces/ITelemetryContext.ts

```diff
@@ -61,4 +61,9 @@ export interface ITelemetryContext {
      * application id obtained from breeze responses. Is used if appId is not specified by root config
      */
     appId: () => string;
+
+    /**
+     * session id obtained from session manager.
+     */
+    sesId: () => string;
```

Comment:
Just noticed this and I'm in 2 minds on whether to leave like this or make it optional.
Why? 
Technically, this is an exposed interface as the ```context``` of an app insights instance and while we don't __expect__ that anyone would provide their own version (to overwrite ours) having this as required make this a breaking change.

Which is where the 2 minds comes from
1) This is an internal interface and people should not be updating this property -- therefore breaking (required method) is fine
2) If there is someone who wants or needs to do this they will be (technically) broken (until they add a function to do this)..

Did we marked the ```context``` accessor to this as readonly?

I'm also thinking about pushing the TypeScript 4 update as the next release which would bump us to at least 2.7 (maybe 3.0) so this change would (I think) be ok for a jump to 2.7 or 3.0 -- lets talk about it in standup tomorrow.

Reply:
What is "context accessor" , I understand this `sesId` is the accessor to session id, and it's not readonly because it can be reassigned right? (eg. each time applySessionContext is called). And sure, let's talk about the update tomorrow.

Reply:
ah, today you can get to the automatic session id via
```appInsights.context.sessionManager.automaticSession```

So what I mean by the ```context accessor``` is the ```context``` property on the ```appInsights``` instance 😄 , it will probably be defined on several interfaces (standard, deprecated and maybe the snippet). There may also be a AppInsightsCore version as well... I suspect it's not currently ```readonly``` (but it probably should be)